### ISSUE TITLE
Question on estimate of exposure-instrument association

### ISSUE THREAD

**Issue by neilstats** [amymariemason/SUMnlmr#20]
In piecewise_summ_mr() with average.exposure.associations = TRUE, the exposure-instrument association appears to be calculated as the inverse-variance weighted average of the stratum-specific estimates (https://github.com/amymariemason/SUMnlmr/blob/main/R/piecewise_new.R#L132). In the nlmr package, the overall estimate is used (https://github.com/jrs95/nlmr/blob/master/R/lace.R#L41). Is this difference because with summary data, _only_ the stratum-specific estimates may be available? If both the overall and stratum-specific estimates are available, is there any reason to prefer using one over the other?

Many thanks for your work, and any guidance you can give is appreciated.

**Comment by amymariemason** [comment]
Good question!
I would expect the meta-analysed strata estimates and the overall estimate to be very similar, if the gene-exposure estimates were indeed constant over the strata (as assumed in the nlmr package, and the residual method in this package). I don't see an obvious reason to prefer one over the other, apart from in SUMnlmr we need the strata-specific estimates as an output anyway so it made sense to use them to give the overall estimate.
If they are giving different answers, then I would be checking that the assumptions on the gene-exposure estimates are holding. 

It is worth noting this was changed from a default setting when we introduced the double-ranked method into the package ( https://www.biorxiv.org/content/biorxiv/early/2022/06/30/2022.06.28.497930.full.pdf ) as that explicitly relaxes the assumption that the gene-exposure association is constant across the strata. Previously we had been assuming that the gene-exposure association was constant, so we could use the estimates from all strata to meta-analyse and get a tighter confidence interval on the gene-exposure estimate.  Removing that assumption meant needing to use strata-specific estimates.  I recommend reading Steve's twitter explanation 
 https://twitter.com/stevesphd/status/1587429116214648832 of this pre-print https://www.medrxiv.org/content/10.1101/2022.10.26.22280570v1 explaining how violations of this can give biased estimates and lead you to incorrect conclusions.
